assets/js/scrollDepth.js

```diff
@@ -0,0 +1,46 @@
+const SCROLL_DEPTH_LEVELS = [20, 40, 60, 80, 99];
+
+window.addEventListener('DOMContentLoaded', function () {
+  let observer
+  const el = document.getElementsByTagName('body')[0];
+    
+  el.style.position = 'relative';
+
+  const initObserver = () => {
+    const cb = (entries) => {
+    const [IntersectionObserverEntry] = entries;
```

Comment:
I don't see `entries` anywhere else, I prefer doing destruction directly in arguments
`( [IntersectionObserverEntry] ) =>` instead

Reply:
sorry I am not following